lenovo/thinkpad/t14

```diff
@@ -0,0 +1 @@
+t14s/
```

Comment:
Can you also add this to README.md and flake.nix?

Reply:
added, can you please check.